### PR TITLE
fix: DEV-3428: Change to 1 item per page changing page by region

### PR DESCRIPTION
**Pull request by juliosgarbi**

### CODE REVIEW

**Review by bmartel — APPROVED** [review]

File: src/tags/object/PagedView.js [hunk 1]
```diff
@@ -82,7 +82,8 @@ const HtxPagedView = observer(({ item }) => {
       if (obj.selected) {
         const _pageNumber = parseFloat(obj.object.name.split('_')[1]) + 1;
 
-        setPage(Math.ceil(_pageNumber / pageSize));
+        changePageSize(1);
```
Comment (bmartel) [inline src/tags/object/PagedView.js:85]: Is this needed if its always just 1 now?

File: src/tags/object/PagedView.js [hunk 2]
```diff
@@ -82,7 +82,8 @@ const HtxPagedView = observer(({ item }) => {
...
+        setTimeout(() => setPage(_pageNumber));
```
Comment (bmartel) [inline src/tags/object/PagedView.js:86]: If you don't reset the pageSize state it wouldn't require a setTimeout here.

File: src/tags/object/PagedView.js [hunk 3]
```diff
@@ -114,6 +115,11 @@ const HtxPagedView = observer(({ item }) => {
     };
   }, [page]);
 
+  const changePageSize = (pageSize) => {
```
Comment (bmartel) [inline src/tags/object/PagedView.js:118]: Not sure the solution here is to hardcode to 1, can you explain the performance issue with the scrolling to?

**Review by bmartel — CHANGES_REQUESTED** [review]

File: src/tags/object/PagedView.js (hunk 1 above)
Comment (juliosgarbi) [reply src/tags/object/PagedView.js:85]: In this case it is change the page size to 1 to show just the item that user choose!

File: src/common/Pagination/Pagination.tsx [hunk 4]
```diff
@@ -13,6 +13,7 @@ interface PaginationProps {
   pageSize: number;
   totalPages: number;
   pageSizeOptions?: [];
+  paginated: boolean;
```
Comment (bmartel) [inline src/common/Pagination/Pagination.tsx:16]: this wording seems odd to parameterize the Pagination component with a `paginated` property. The definition of the component is that it is `paginated` always, but in this case we are changing the ability to select the `pageSize` not enabling/disabling the entire component. Maybe we change `paginated` to something like `pageSizeSelectable` as this is the only affected portion of the component.

**Review by bmartel — CHANGES_REQUESTED** [review]

File: src/common/Pagination/Pagination.tsx (hunk 4 above)
Comment (juliosgarbi) [reply src/common/Pagination/Pagination.tsx]: it makes sense!

**Review by bmartel — APPROVED** [review]

**Pull request merged**